pkg/cloudprovider/aws/suite_test.go

```diff
@@ -75,6 +76,7 @@ var _ = BeforeSuite(func() {
 					securityGroupProvider: NewSecurityGroupProvider(fakeEC2API),
 					cache:                 launchTemplateCache,
 				},
+				cache.New(5*time.Second, 30*time.Second),
```

Comment:
If you export these values you can reference them directly in the tests. Assuming they're not different.

Reply:
You'll also likely need to use injectable.Time to test this, though.